data/common.yaml

```diff
@@ -22,3 +22,5 @@ icinga::agent::logging_level: warning
 icinga::agentless::user: icinga
 icinga::agentless::manage_user: true
 icinga::agentless::ssh_key_type: rsa
+
+icinga::cert_name: "%{facts.networking.fqdn}"
```

Comment:
I don't quite understand this change. Do you do this because you want to use `icinga::cert_name` within other Hiera values?

Reply:
correct, like in a profile class for icinga objects
profile::icinga::objects:
  Host:
    "%{lookup('icinga::cert_name')}"
      attribute1: value1
      attribute2: valie2

Reply:
Like here, https://github.com/lbetz/icinga-summit-2024/blob/5598e34b03ace959c598413b0e24e1e313a0dde4/puppetserver/code/environments/production/data/common.yaml#L11